### PR TITLE
Pbm 429 log backup progress

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+Run |pbm-speed-test-storage-help| for the full set of available flags:
+
+.. include:: .res/code-block/bash/pbm-speed-test-storage-help.txt
+
+Backup progress logs
+============================================================================
+
+Track backup progress in |pbm-agent| logs. Log messages are written
+every minute.
```
Comment (akira-kurogane) [inline doc/source/troubleshooting.rst:64]:
```suggestion
If you have a large backup you can track backup progress in |pbm-agent| logs. A line is appended every 
minute showing bytes copied vs. total size for the current collection.
``

File: doc/source/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Compression test
+----------------------------------------------
+
+.. include:: .res/code-block/bash/pbm-speed-test-compression.txt
+
+|pbm-speed-test-compression| uses the compression library from the config
+file and sends a fake semi random data document (1 GB by default) to the
+black hole storage. (Use the ``pbm config`` command to change the compression library). 
+After the test, the data is removed.
```
[the remark this answers is not in the document]
Comment (akira-kurogane) [reply doc/source/troubleshooting.rst:38]: The "pbm-speed-test storage" test does create data which will be deleted after though - I guess it should be written there.

[the remark this answers is not in the document]
Comment (nastena1606) [reply doc/source/troubleshooting.rst:38]: After conversation with @dAdAbird I've got the impression that the data created during the pbm-speed-test storage remains in the storage.

[the remark this answers is not in the document]
Comment (akira-kurogane) [reply doc/source/troubleshooting.rst:38]: Ah, my misunderstanding about it [not] being deleted after "pbm-speed-test storage".

But it's never persisted anywhere for the compression test, so let's remvoved this line from the pbm-speed-test compression section.

**Pull request merged**